src/main/webapp/app/text-assessment/text-assessment.component.html

```diff
@@ -17,7 +17,7 @@ <h5 class="text-white">
             <button
                 class="btn btn-primary ml-3"
                 (click)="save()"
-                [disabled]="!assessments || assessments.length === 0 || !assessmentsAreValid || !isAuthorized"
```

Comment:
The check is covered in `assessmentsAreValid`

Reply:
But when there is no feedback `assessmentsAreValid` will be set to true in `checkScoreBoundaries()`, i.e. saving an empty assessment is possible. Or am I missing something?

Reply:
I overlooked something and will rework the validation.

Reply:
I would just check for `referencedFeedback` in the `checkBoundaries()` method. If there is no `referencedFeedback` the assessment is not valid, otherwise check for the score of each feedback. Similar to `validateFeedback()` in the `ModelingAssessmentEditorComponent` just additionally calculating the total score.